src/Chapter1.hs

```diff
@@ -631,9 +636,12 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int
+sumLast2 n = secondLastDigit n + lastDigit n -- lastDigit was defined above
```

Comment:
Great reusage of the `lastDigit` function here 💯

Reply:
Thank you!